test/test_runner.go

```diff
@@ -259,8 +272,25 @@ func (r *TestRunner) RunTests(script string) (results Results, err error) {
 		return nil, err
 	}
 
+	err = r.runTestInit(inter)
+	if err != nil {
+		return nil, err
+	}
+
+	script = replaceImports(program.Program, script)
+	program, inter, err = r.parseCheckAndInterpret(script)
```

Comment:
Instead of calling again `r.parseCheckAndInterpret(script)` for the new script, maybe we can just do:

```go
program, err := r.testRuntime.ParseAndCheckProgram([]byte(script), ctx)
if err != nil {
	return nil, nil, err
}

_, inter, err := env.Interpret(
	ctx.Location,
	program,
	nil,
)
```

by using the already created `env` & `ctx`.